lib/order.rb

```diff
@@ -0,0 +1,65 @@
+class Order
+
+  attr_reader :current_order, :bill, :string_summary
+
+  def initialize
+    @menu = Menu.new
+    @current_order = []
+    @bill = 0
+    @string_summary = ''
+  end
+
+  def order_received(food_order, quantity = 1 )
+    raise "There is no #{food_order} available on the menu" unless on_menu?(food_order)
```

Comment:
Having an `on_menu?` helper method makes the code nice and clear! Does the method belong on the order?